pydantic_ai_slim/pydantic_ai/ui/_web/api.py

```diff
@@ -0,0 +1,114 @@
+"""API routes for the web chat UI."""
+
+from collections.abc import Sequence
+from typing import Any
+
+from pydantic import BaseModel
+from pydantic.alias_generators import to_camel
+from starlette.applications import Starlette
+from starlette.requests import Request
+from starlette.responses import JSONResponse, Response
+
+from pydantic_ai import Agent
+from pydantic_ai.builtin_tools import AbstractBuiltinTool
+from pydantic_ai.toolsets import AbstractToolset
+from pydantic_ai.ui.vercel_ai import VercelAIAdapter
+
+
+class ModelInfo(BaseModel, alias_generator=to_camel, populate_by_name=True):
+    """Defines an AI model with its associated built-in tools."""
+
+    id: str
+    name: str
+    builtin_tools: list[str]
+
+
+class BuiltinToolInfo(BaseModel, alias_generator=to_camel, populate_by_name=True):
+    """Serializable info about a builtin tool for frontend config."""
+
+    id: str
+    name: str
+
+
+class _ConfigureFrontend(BaseModel, alias_generator=to_camel, populate_by_name=True):
```

Comment:
Now that the module is private, the class can lose the `_`. Double check this in the other private modules as well please